source/common/buffer/watermark_buffer.cc

```diff
@@ -1,5 +1,4 @@
 #include "source/common/buffer/watermark_buffer.h"
-#include "watermark_buffer.h"
```

Comment:
hm, so before in this case same header was included twice?

Reply:
#pragma once will ensure the content won't be repeated

Reply:
right, missed that one :)